playbooks/adjust-cinder-backend.yml

```diff
@@ -111,6 +117,11 @@
 
   - name: trigger stop all cinder services
     meta: flush_handlers
+    
+
+  - name: wait for cinder services down
+    shell: sleep 10s
+    when: set_cinder_host or update_cinder_backend
```

Comment:
I tried check-cinder-service.sh under sensu plugin dir. it didn't work very well.
the sensu check can make sure service is up, while it can promise `cinder type-creat` works.
I use `cinder list` instead